lib/LedgerSMB/Template.pm

```diff
@@ -278,14 +278,9 @@ sub _preprocess {
     my ($rawvars, $escape) = @_;
     return undef unless defined $rawvars;
 
-    { # pre-5.14 compatibility block
-      # Really? -YL
-        local ($@); # pre-5.14, do not die() in this block
-        if (eval {$rawvars->can('to_output')}){
-            $rawvars = $rawvars->to_output;
-        }
+    if (eval {$rawvars->can('to_output')}){
```

Comment:
This is now missing a

```perl
    local ($@);
```
The thing about the compat block is that the local has a tightly restricted scope, which is no longer required.